docs/xjit.md

```diff
@@ -62,6 +62,7 @@ These parameters can be used to modify the behavior of `-Xjit`:
 | [`optlevel`      ](#optlevel      ) | Forces the JIT compiler to compile all methods at a specific optimization level.        |
 | [`verbose`       ](#verbose       ) | Reports information about the JIT and AOT compiler configuration and method compilation.|
 | [`vlog`          ](#vlog          ) | Sends verbose output to a file.                                                         |
+| [`perfTool`          ](#perftool          ) | Generates `/tmp/perf-<pid>.map` file for JIT compiled code for profiling the VM.                                                         |
```

Comment:
```suggestion
| [`perfTool`          ](#perftool          ) | Generates the `/tmp/perf-<pid>.map` file to facilitate JIT compiled code symbol resolution when profiling the VM using `perf` on Linux.                                                         |
```
Also maybe it's worth removing all the extra whitespace.

Reply:
I have removed some whitespaces but retained some to maintain the readability in the markdown as is the practice.

I have also made the sentence shorter to make it consistent with the other description in this table.